### ISSUE TITLE
Feature : Make API conform to web push standard on iOS n Android

### ISSUE THREAD

**Issue by awebdeveloper** [phonegap/phonegap-plugin-push#1614]
### Expected Behaviour

Web Push API code should work on iOS n Android albeit with minor changes. I found this but looks like no activity has happened during the last 2 years.

https://github.com/MobileChromeApps/cordova-plugin-service-worker-push/blob/master/README.md

**Comment by macdonst** [comment]
@awebdeveloper yes, it is my intention to move the plugin to the Web Push API. No timeframe as of yet but hoping it will be by mid this year.

**Comment by awebdeveloper** [comment]
Great. If you can list down what all are needed. I can see if some one can help expedite the process.

**Comment by macdonst** [comment]
@awebdeveloper 

**Step 1:** look at where the API differs from the current standard
**Step 3:** make it work

but seriously if I had it all planned out I'd probably have done it by now :)

**Comment by awebdeveloper** [comment]
Ha Ha. Lets see if i can atleast create that doc of what all are the diff b/w the two.

**Comment by filmaj** [comment]
We could probably come up with a doc to detail the differences and clarify an implementation path by perusing this repo: https://github.com/w3c/push-api
